packages/corvid-cli/src/utils/electron.js

```diff
@@ -152,14 +153,29 @@ async function openWindow(app, windowOptions = {}) {
       setTimeout(resolve, isDevTools ? 1000 : 0);
     }).then(async () => {
       const corvidConfig = await readCorvidConfig(".");
-
+      win.webContents
+        .executeJavaScript(
+          "(() => ({santaBase: window.santaBase, editorBase: window.editorBase}))()"
+        )
+        .then((result = {}) => {
+          const getVersion = (base = "") => {
```

Comment:
simpler version of the function
const getVersion = (base = "") => base.substring(base.lastIndexOf('/') + 1)

Reply:
done